src/main/java/carpet/script/language/ControlFlow.java

```diff
@@ -145,5 +151,23 @@ public static void apply(Expression expression) // public just to get the javado
                 return (c_, t_) -> retval;
             }
         });
+
+        expression.addUnaryFunction("encode_b64", v -> StringValue.of(Base64.getEncoder().encodeToString(v.getString().getBytes())));
```

Comment:
ControlFlow is for controlling flow of programs not for IO utils. Use Sys or DataStructures

Reply:
k